### ISSUE TITLE
Tenant Subdomains

### ISSUE THREAD

**Issue by bahrmichael** [jeremydaly/reference-architectures#2]
To give customers their own branded websites, while keeping the source code lean, I've used the following approach a couple times:

![tenant subdomain](https://user-images.githubusercontent.com/1830132/86408370-2be9c880-bcb7-11ea-8164-044c09a5df40.png)

Everything on AWS (Route53, Certificates, CloudFront, S3, APIGW), just the CMS is on Contentful right now. CMS can be skipped if you just want convenient subdomains.

Do you think this is a pattern worth exploring further for the reference architectures?

**Comment by jeremydaly** [comment]
Definitely. Can you submit a more detailed architecture?

**Comment by bahrmichael** [comment]
Yes, I'm currently working on a blog post which also covers this topic. Once I figured out what problem exactly this is good for and have a rough draft, I'll add more details here :)
